### DISCUSSION TITLE
Change title in Conservative staking

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#13] [Bugs and Reports]
change title which is on top of your revenues from "conservative staking", to "Your revenues". Same as when you see in dynamic staking. 
<img width="448" alt="image" src="https://github.com/user-attachments/assets/a6721a16-85f5-4c11-a2ac-149aa474a4f0">

**Comment by bf-monster** [comment]
noted. will be fixed

**Reply by bf-monster** [reply]
fixed
